packages/envd/internal/api/upload.go

```diff
@@ -110,7 +110,7 @@ func processFile(r *http.Request, path string, part *multipart.Part, user *user.
 func resolvePath(part *multipart.Part, paths *UploadSuccess, u *user.User, params PostFilesParams) (string, error) {
 	var pathToResolve string
 
-	if params.Path != nil {
+	if params.Path != nil && *params.Path != "" {
```

Comment:
I think we would ideally make it so that the SDK does not need this change.

Reply:
This is adapting to the SDK (the `js-sdk` at least), as in, in the SDK you can only send `string` not `string | undefined`

Reply:
I would like to know if the API error is caused by the client generated from the OpenAPI spec or if this is because of the envd implementation.

Reply:
it comes from `handleEnvdApiError`

```
 ❯ Module.handleEnvdApiError src/envd/api.ts:20:14
     18|   switch (res.response.status) {
     19|     case 400:
     20|       return new InvalidArgumentError(message)
       |              ^
     21|     case 401:
     22|       return new AuthenticationError(message)
```

Reply:
it says it's optional from the OpanAPI spec: https://github.com/e2b-dev/infra/blob/main/packages/envd/internal/api/api.gen.go#L235